node/test/helpers.ts

```diff
@@ -8,10 +8,13 @@ import yn from 'yn';
 import axios from 'axios';
 import { Express } from 'express';
 import request from 'supertest';
+import MockAxios from 'axios-mock-adapter';
 
 export const MOCKING_DISABLED = yn(process.env.MOCKING_DISABLED || 0);
 export const DIALOG_ENDPOINT = process.env.DIALOG_ENDPOINT || '/dialog';
 
+export const MOCK_AXIOS = new MockAxios(axios);
```

Comment:
seems weird and possibly dangerous to do this at the static/module level as opposed to inside the scope of a test as it has been up until now...do we really need this?

Reply:
when I tried to use separate mockAxios instances, only one file would properly return the mock responses.  I couldn't figure out why it was doing that, so my current solution was to use only one instance.  If you can figure out what's going on there please let me know.